cloudinit/net/ifconfig.py

```diff
@@ -0,0 +1,22 @@
+# This file is part of cloud-init. See LICENSE file for license information.
+
+import copy
+import glob
+import logging
+import os
+import re
+from typing import Optional
+
+from cloudinit import subp, util
+from cloudinit.net import ParserError, renderer, subnet_is_ipv6
+from cloudinit.net.network_state import NetworkState
+
+LOG = logging.getLogger(__name__)
+
+def available(target=None):
+    expected = ["ifconfig"]
+    search = ["/sbin"]
+    for p in expected:
```

Comment:
this loop can be eliminated. And honestly, maybe the `which(ifconfig)`, too ;)
We might just check `if is_BSD()` and be done with it

Reply:
I'll remove the loop, but leave the `which(ifconfig)` for now